languages/python/exercises/concept/comments/.docs/introduction.md

```diff
@@ -0,0 +1,5 @@
+Comments are part of every programming language. They provide important information about the program to its stakeholders. Every programmer has a different habit of logging comments in the program. Some organizations insist programmers log little detail as possible. So the next programmers would not get confused with the previous person's work.
+Comment is part of the program where part of the information is omitted by the compiler in the compilation.
+Organizations prefer to put build date, contributor’s name, deployment date in their comments. A good programmer will log little details as possible so he won't get confused when complexity increases in programming.
```

Comment:
Change to "as little detail as possible"

Also, I'm unsure of how truthful this statement is.